mapping_parenting_tech/analysis/prototyping/app_store/app_output.json

```diff
@@ -0,0 +1,929 @@
+{
```

Comment:
I guess can keep this json here as an example, but generally we should keep the output files away from code, in the outputs folders (which then can be synced with s3)

Reply:
Will delete.